packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1224,25 +1224,16 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
-    _maybeTriggerScrollbar();
+    assert(_debugScheduleCheckHasValidScrollPosition());
```

Comment:
Won't this result in the scrollbar not being triggered in release builds? Asserts are disabled in release.